zerver/lib/actions.py

```diff
@@ -906,6 +907,44 @@ def do_send_messages(messages):
     # intermingle sending zephyr messages with other messages.
     return already_sent_ids + [message['message'].id for message in messages]
 
+def add_reaction(user_profile, message, emoji):
+    # type: (UserProfile, Message, RealmEmoji) -> None
+    reaction = Reaction()
+    reaction.sender = user_profile
+    reaction.message = message
+    reaction.emoji = emoji
+    reaction.save()
+
+    sender = {'id': user_profile.id, 'email': user_profile.email}
+
+    event = {'type': 'reaction',
+             'sender': sender,
```

Comment:
I think we may want to the put the sender's `full_name` in here for analogy with how other events work.  Also, call it `user_id`, not `id`; that's the new convention.